doc/examples/cloud-config-user-groups.txt

```diff
@@ -7,6 +7,11 @@ groups:
   - cloud-users
 
 # Add users to the system. Users are added after groups are added.
+# Note: Following configurations in the below examples are not
+#       applicable on already existing user(s).
+#       - 'plain_text_passwd', 'hashed_passwd', 'lock_passwd', 'sudo',
+#         'ssh_authorized_keys', 'ssh_redirect_user',
+#         'cloud_public_ssh_keys'.
```

Comment:
Similarly, it should be dropped from here:
```suggestion
#         'ssh_authorized_keys', 'ssh_redirect_user'
```

Reply:
Thanks for the inputs, I have pushed one more set of changes.